### ISSUE TITLE
Add a keyboard shortcut to toggle Kubectl Shell

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#4736]
**Detailed Description**
Add a keyboard shortcut to toggle the Kubectl shell when it is available. VSCode uses Ctrl+` by default, maybe something similar will work well.

**Context**
It would be a simple QOL improvement to be able to press a key combo to raise the kubectl shell and start typing.

**Comment by nwmac** [comment]
This should be fairly straight forward.

This would be for the explorer view show below.

We already have a keyboard shortcut for the search button, so this is adding the same for the kubectl shell.

Look in `components/nav/Header.vue` at how we do `searchShortcut` - need to do the same for the Kubernetes shell button.

![image](https://user-images.githubusercontent.com/1955897/150499892-bb1ffb17-5c9c-4b97-a572-92fd2e63b996.png)

**Comment by vincent99** [comment]
There is [already](https://github.com/rancher/dashboard/blob/6563094169c26aa2f7087e2e125c684a34494f14/layouts/default.vue#L612) a key for this.

**Comment by rak-phillip** [comment]
Maybe we can add a hint for the keyboard shortcut to the popover like we do for Resource Search? 

I'm also thinking that it would be a good idea to take a closer look at #2982 so we can make our existing shortcuts a little more discoverable.

**Comment by nwmac** [comment]
Yeah we should and we should add a page with the shortcuts we have and look at what others we should add.

**Comment by sgapanovich** [comment]
### Test Environment: ###
Rancher version: v2.6-head bc8260d
Rancher cluster type: single-node docker install
Docker version: 20.10

Downstream cluster type: ec2 node driver

----------

### Testing: ###

when hovering over the "Kubectl Shell" button -> correct tooltip shown in both languages
![label](https://user-images.githubusercontent.com/40282141/151393828-a1f170b2-798f-40e1-bf3b-680c7d31664c.jpg)


using a short cut triggers the same actions as clicking on the button:
-"dashboard-shell-xxxx" pod is created in the cattle-system namespace (per each shell opened)
-kubectl shell is opened for the current cluster (tested in local and downstream)
-same events logged (

"Ctrl + `" works for both Mac and Windows
back tick also is a shortcut for the kubectl shell (as Vincent mentions)
